src/components/meta/VButton.stories.mdx

```diff
@@ -0,0 +1,120 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs'
+
+import VButton, {
+  buttonForms,
+  buttonSizes,
+  buttonVariants,
+} from '~/components/VButton.vue'
+
+<Meta
+  title="Components/VButton"
+  components={VButton}
+  argTypes={{
+    as: {
+      options: buttonForms,
+      control: { type: 'radio' },
+    },
+    variant: {
+      options: buttonVariants,
+      control: { type: 'select' },
+    },
+    pressed: { control: 'boolean' },
+    size: {
+      options: buttonSizes,
+      control: { type: 'radio' },
+    },
+    disabled: { control: 'boolean' },
+    focusableWhenDisabled: { control: 'boolean' },
+    type: { control: 'text' },
+  }}
```

Comment:
Storybook appears to treat these args as the "default" set that are present in all stories. This means that the variant and `as` arg are still present on the ButtonVariants story. I guess we'd need to move these down to the Canvas for the first story to prevent that.

Reply:
Thank you for catching this! I didn't notice